oc-game-swift.xcodeproj/xcuserdata/noox.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -9,6 +9,11 @@
 			<key>orderHint</key>
 			<integer>0</integer>
 		</dict>
+		<key>oc-game-swift.xcscheme</key>
+		<dict>
+			<key>orderHint</key>
```

Comment:
why do you add this variable in this `plist` ?

Reply:
I think xcode added that when i've changed the name of the project.